internal/utils/misc.go

```diff
@@ -38,7 +38,7 @@ const (
 	ImageProxyImage  = "darthsim/imgproxy:v3.8.0"
 	EdgeRuntimeImage = "supabase/edge-runtime:v1.53.3"
 	VectorImage      = "timberio/vector:0.28.1-alpine"
-	PgbouncerImage   = "bitnami/pgbouncer:1.20.1-debian-11-r39"
+	SupavisorImage   = "supabase/supavisor:1.1.56"
```

Comment:
Out of curiosity, why not use the latest version here? Does it mean that the pooler team should constantly update the version in this repo? For example, the RT version is too old (April 3) 

cc @filipecabaco

Reply:
We can't use `latest` tag in our default registry ECR because it requires immutable tags.

Typically other services like auth and storage expose an endpoint that returns the currently deployed version string. CLI queries that endpoint to update user's local environment. This way pooler team doesn't need to constantly update the version here.

When there's major features / releases, it's typically better to bump the version manually so it's more explicit when we deprecate certain configs, like statement mode, or add new config options to support new features.

Reply:
> Typically other services like auth and storage expose an endpoint that returns the currently deployed version string

Supavisor can provide this as well

```bash
$ curl -s -i http://aws-0-eu-west-3.pooler.supabase.com:5432 | grep 'x-app-version'
x-app-version: 1.1.49
```

Reply:
That's nice, I will make a separate PR to sync the pooler version.